.github/workflows/cherry-picking.yaml

```diff
@@ -0,0 +1,38 @@
+name: Cherry picking
+on:
+  push:
+    branches:
+      - master
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    env:
+      #Variable for the name of the branch to cherry-pick into.
+      #It will remain 'invalid' if no branch is specified
+      destination_branch: 'invalid'
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    #Removes posible conflicting characters on the commit message
```

Comment:
conflicting characters are？

Reply:
In this case, quotation marks, this is because the content of the message will be passed to a script as a parameter (line 28) and quotation marks will split the text as if it where multiple parameters.

Reply:
Thanks, please put this as comment below this line. After that we can merge this and give it a try.